Tools/CISupport/ews-build/events.py

```diff
@@ -311,8 +311,9 @@ def stopService(self):
 
 
 class GitHubEventHandlerNoEdits(GitHubEventHandler):
-    ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
```

Comment:
Why is this being removed?

Reply:
Seems like buildbot is also checking for reopened:
reference:https://github.com/buildbot/buildbot/blob/master/master/buildbot/www/hooks/github.py#L190